README.md

```diff
@@ -1,56 +1,42 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Tour Booking App (backend)
```

Comment:
- [ ] On the license section of your readme, the link is broken.  Kindly fix the link.

## Explanation

The link is broken because it points to a file that does not exist.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file, copy it to the root of your project and link to it in your "license" section of your readme.

That way, your app will be well documented 💪